ft-email-auth-provider/src/handlers/create_account.rs

```diff
@@ -1,3 +1,89 @@
+/// Create account handler, this is available on /create-account/ route
+///
+/// If a user does not exist for the given username and email, we create it.
+///
+/// It can happen that a user exists for the given email, but has no identity
+/// (identity is empty string). This can happen if you have imported email
+/// subscribers to the user table.
+///
+/// When importing, make sure that only unverified email is added in the
+/// email provider data, nothing else. E.g. `data -> 'email'` should only
+/// contain `{ "emails": ["email@being-imported.com"] }`, all other
+/// subscriber data, e.g. if there is double opt-in, or the `name` of user,
+/// `tags` for the user should be stored in any other `data` key (`data -> 'subscriptions')
+#[ft_sdk::form]
+pub fn create_account(
+    mut conn: ft_sdk::Connection,
+    ft_sdk::Form(payload): ft_sdk::Form<CreateAccountPayload>,
+    ft_sdk::Cookie(sid): ft_sdk::Cookie<{ ft_sdk::auth::SESSION_KEY }>,
+    // code can be invalid. eg: xyz
+    ft_sdk::Query(code): ft_sdk::Query<"code">,
```

Comment:
This line will break the normal create account flow. Convert this to optional type.

Reply:
Fixed this